manifests/extension/runkit.pp

```diff
@@ -0,0 +1,58 @@
+# Installs a php extension for a specific version of php.
+#
+# Usage:
+#
+#     php::extension::runkit { 'runkit for 5.4.10':
+#       php     => '5.4.10',
+#       version => '1.0.3'
+#     }
+#
+define php::extension::runkit(
+  $php,
+  $version = '1.0.3'
+) {
+  include boxen::config
+
+  require php::config
+  # Require php version eg. php::5_4_10
+  # This will compile, install and set up config dirs if not present
+  php_require($php)
+
+  $extension = 'runkit'
+
+  # Final module install path
+  $module_path = "${php::config::root}/versions/${php}/modules/${extension}.so"
+
+  # Clone the source respository
+  repository { "${php::config::extensioncachedir}/runkit":
+    source => 'namesco/runkit'
```

Comment:
I'm not a fan of using non-canonical source. Can you get the upstream tagged by the original author?